### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -85,6 +85,32 @@
...
+		path="/path"
+		with_sftp=0
+		password="myreallystrengthpassword"
+		is_public=1
+		phpversion="8.2"
+		with_mysql=0
+	; Checks
+		setup_sub_dir=1
+		upgrade=1
+		backup_restore=1
```
Comment (alexAubin) [inline check_process:113]: Hmmm may I encourage to use `tests.toml` ? :sweat_smile:

**Comment by alexAubin** [comment]
Sooooooo https://ci-apps-dev.yunohost.org/ci/job/5370 pretty much worked, there are glitches about the witness files unrelated to the app, I would just ignore those and merge the PR in its current state

**Review by alexAubin — APPROVED** [review]

**Pull request merged**